libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -345,7 +353,8 @@ bool ASTJsonConverter::visit(FunctionDefinition const& _node)
 		make_pair("kind", TokenTraits::toString(_node.kind())),
 		make_pair("stateMutability", stateMutabilityToString(_node.stateMutability())),
 		make_pair("visibility", Declaration::visibilityToString(_node.visibility())),
-		make_pair("overrides", _node.overrides() ? toJson(_node.overrides()->overrides()) : Json::nullValue),
+		make_pair("virtual", _node.markedVirtual()),
+		make_pair("overrides", _node.overrides() ? toJson(*_node.overrides()) : Json::nullValue),
```

Comment:
This change with implementing the visitor feels like a breaking change on how the overrides are encoded?

Reply:
That could very well be. Previously they didn't provide enough info to recreate the AST

Reply:
I mean, there is a reason this is a PR to 060